ocs_ci/utility/aws.py

```diff
@@ -262,6 +257,26 @@ def detach_and_delete_volume(self, volume, timeout=120):
             delete_response
         )
 
+    def wait_for_instance_status(self, instance_id, status, timeout=TIMEOUT):
+        """
+
+        Args:
+            instance_id (str): ID of the instance to stop
+            status (int): The status code to wait for
+            timeout (int): The number in seconds to wait for the instance
+                reach the status
+
+        Returns:
+            bool: True in case the instance reached the status, False otherwise
+
+        """
+        for sample in utils.TimeoutSampler(
```

Comment:
Lets use boto waiters instead of function sampler - https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.Waiter.InstanceRunning

Reply:
@vasukulkarni Nice!
Also found these 2:
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.Instance.wait_until_running
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.Instance.wait_until_stopped

Reply:
👍